editor/components/inserter/style.scss

```diff
@@ -162,11 +173,24 @@ $block-inserter-search-height: 38px;
 
 .editor-inserter__item-has-children {
 	.editor-inserter__item-icon {
-	    box-shadow: 4px 4px rgba( $dark-gray-900, 0.3 );
 		margin-right: 3px;
 		position: relative;
 		top: -2px;
 		left: -2px;
+
+		// Show a "stacked card" below an item that has children.
```

Comment:
This is why I CC'ed you for review, @jorgefilipecosta — I know you're working on a color thing in #7068, but in this PR I'm changing so that the colored car below the topmost is not a `box-shadow`, but a separate pseudo element. Can you work with this?

Reply:
Thank for the ping @jasmussen. I did some research and unfortunately, we don't have a clean way to set styles for pseudo-elements like :after dynamically.
The options we have are creating a style element dynamically dangerouslySetInnerHTML and creating an empty element (that acts like the after ) where we apply the styles. I think this second option although not great is acceptable.

Reply:
Hmm. Like an extra empty dummy element in a block that has children? That could work.

But can you elaborate on why we can't set styles dynamically? Wouldn't you just output some CSS like this? 

```
.woo.editor-inserter__item-icon:after {
	background: purple;
}
```

Where ".woo" would be the classname of whatever branded block is supplying the colors.

Reply:
We would need to output the styles itself dynamically not just pass a class something like:
```
.editor-inserter__item-icon:after {
	background: ${ color-set-by the-block;
}
```

The way to output the styles dynamically with React would be using `<style dangerouslySetInnerHTML={{...`, and using dangerouslySetInnerHTML in React should be avoided if possible.

Reply:
Creating an empty element if the block has children seems the way to go.

Reply:
@jorgefilipecosta I pushed a change in https://github.com/WordPress/gutenberg/pull/7074/commits/03894bd3b49d9497613cfeb983447c8840f7229d to use a separate element instead. Did I do it right, and can you work with that? Thanks.